agent/bin/test_install

```diff
@@ -72,3 +72,8 @@ git clone --depth 1 https://github.com/bats-core/bats-core.git \
     && cd bats-core \
     && ./install.sh "$BATS_DIR"
 )
+
+export BASHUNIT_DIR=$PWD/build/bashunit
+mkdir -p $BASHUNIT_DIR
+curl -s https://bashunit.typeddevs.com/install.sh | bash -s $BASHUNIT_DIR
+sed -ibak 's;#!/bin/bash;#!/usr/bin/env bash;' $BASHUNIT_DIR/bashunit
```

Comment:
The change to use the version of bash specified on the PATH has been accepted by the bashunit devs. I'll remove this hack once it's been released.